module_utils/network_lsr/argument_validator.py

```diff
@@ -246,6 +246,54 @@ def _validate_string_min_length(self, value):
             return True
 
 
+class ArgValidatorRouteTable(ArgValidator):
+    def __init__(
+        self,
+        name,
+        required=False,
+        default_value=None,
+    ):
+        ArgValidator.__init__(self, name, required, default_value)
+
+    def _validate_impl(self, value, name):
+        v = None
+        val_min = None
+        val_max = None
+        try:
+            if isinstance(value, bool):
+                # bool can (probably) be converted to self.numeric_type,
+                # but here we don't want to accept a boolean value.
+                pass
+            elif isinstance(value, int):
+                val_min = 1
+                val_max = UINT32_MAX
+                v = int(value)
+            elif isinstance(value, Util.STRING_TYPE):
+                try:
+                    v = int(value)
+                except Exception:
+                    v = value
```

Comment:
I think you should check that the string contains only a benign set of characters and that it's not empty. An alias should be a plain word, no spaces, or newlines.